src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -1247,17 +1259,14 @@ public void visitSources(SCMSourceObserver observer) throws IOException, Interru
     }
   }
 
-  private Iterable<GHRepository> getGhRepositoriesBasedOnTopics(
-      final TaskListener listener, final GitHub github, final List<String> topics) {
-    listener
-        .getLogger()
-        .println(
-            GitHubConsoleNote.create(
-                System.currentTimeMillis(),
-                String.format("Looking up repositories for topics: '%s'", topics)));
+  private Iterable<GHRepository> searchRepositories(
+      final GitHub github, final GitHubSCMNavigatorContext context) {
     final GHRepositorySearchBuilder ghRepositorySearchBuilder = github.searchRepositories();
-    topics.forEach(ghRepositorySearchBuilder::topic);
-    ghRepositorySearchBuilder.q("org:" + getRepoOwner());
+    context.getTopics().forEach(ghRepositorySearchBuilder::topic);
+    ghRepositorySearchBuilder.org(getRepoOwner());
```

Comment:
Thanks for updating this :)

I had put a pull request into github api a while ago

https://github.com/hub4j/github-api/pull/1355